lms/urls.py

```diff
@@ -108,10 +108,9 @@
 
     # URLs for API access management
     url(r'^api-admin/', include('openedx.core.djangoapps.api_admin.urls', namespace='api_admin')),
-)
 
-urlpatterns += (
     url(r'^dashboard/', include('learner_dashboard.urls')),
+    url(r'^api/experiments/v0/', include('experiments.urls')),
```

Comment:
Nit: in terms of isolation, it's better to have the djangoapp handle its own versioning - so the `v0` part would be moved into the app's own urls.py.

But it's not a release blocker for our experiments.